File: LoopConstruction/C/power.c
#include <stdio.h>

int main() {
    int x, y;
    long long result = 1;

    printf("Enter the value of x and y: ");
    scanf("%d %d", &x, &y);

    for(int i = 1; i <= y; i++) {
        result *= x;
    }

    printf("%d raised to the power %d is %lld\n", x, y, result);

    return 0;
}

File: LoopConstruction/Java/power.java

import java.util.Scanner;

public class power {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter base and power");
        int base = sc.nextInt();
        int power = sc.nextInt();
        long result = 1;

        for(int i = 0;i<power; i++){
            result = result * base;
        }
        System.out.println(base+" raised to the power "+power+" is "+result);
    }
}
